invenio_rest/errors.py

```diff
@@ -91,6 +92,9 @@ def get_body(self, environ=None):
         if self.errors:
             body['errors'] = errors
 
+        if self.code >= 500 and hasattr(g, 'sentry_event_id'):
+            body['internal_error_id'] = str(g.sentry_event_id)
```

Comment:
Perhaps just ``error_id`` instead? In user facing templates i think we call it "Error identifier"